app/controllers/room_bookings_controller.rb

```diff
@@ -19,14 +19,28 @@ def index
       section: [:course],
       time_slot: {},
       instructor: {}
-    ).where(rooms: { schedule_id: }, time_slots: { day: @active_tab })
+    ).where(rooms: { schedule_id: @schedule.id }, time_slots: { day: @active_tab })
```

Comment:
we already have schedule_id saved, so why are we getting the schedule and then its ID?

Reply:
We don't have schedule_id anymore, it's been deleted to use the common functionality.